src/components/single-line-diagram.js

```diff
@@ -37,6 +37,8 @@ const SingleLineDiagram = (props) => {
 
     const dispatch = useDispatch();
 
+    const useName = useSelector(state => state.useName);
```

Comment:
This component does not need to have a state, title should be passed as a property.